core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -320,20 +323,20 @@ import com.datastax.driver.core.utils.UUIDs
               s"lookingForMissingCalled for tag ${session.tag} when there " +
               s"is no missing. Raise a bug with debug logging.")
         }
-
         if (missing.deadline.isOverdue()) {
-          log.info(
-            "[{}] [{}]: Failed to find missing sequence nrs. Missing info: [{}]",
```

Comment:
:+1: I guess it wasn't meant to be a user message anyway. Too many negation inside :)